qiskit/providers/aer/quantum_info/states/__init__.py

```diff
@@ -0,0 +1,19 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+=============================================
+State (:mod:`qiskit.providers.aer.quantum_info.state`)
```

Comment:
I think you're missing a file for this path to work. There should be a `qiskit/providers/aer/quantum_info/__init__.py` too to make this subpackage importable. We might also want to re-export `AerStatevector` and `AerState` off that root quantum info path too.